saleor/graphql/core/tests/test_graphql.py

```diff
@@ -59,7 +59,7 @@ def test_jwt_middleware(client, admin_user):
     response = api_client_post(data={"query": create_token_query})
     repl_data = response.json()
     assert response.status_code == 200
-    assert response.wsgi_request.user == admin_user
+    assert isinstance(response.wsgi_request.user, AnonymousUser)
```

Comment:
I think the original logic is slightly backward - we send anonymous request with admin credentials to create new token for admin and then we check the response if its request's user is the admin we created the token for. In my opinion the user should still be anonymous or should not matter. The user was there because we had old login mechanics which this PR modifies and/or removes. In next lines we use the generated token to succesfully authenticate the admin so this assertion shouldn't even be here because it does not matter. If I'm wrong and this users has to be admin, please provide a better usecase than this test.